### PR TITLE
Update the package.xml files with the latest Open Robotics maintainers

### PR DESCRIPTION
**Pull request by mjeronimo**
See title :)

Signed-off-by: Michael Jeronimo <michael.jeronimo@openrobotics.org>

### CODE REVIEW

**Review by clalancette — CHANGES_REQUESTED** [review]
In general, when we remove maintainers we tend to move them to an `author` tag.  That's just to acknowledge the work they've done in the past.

**Comment by clalancette** [comment]
You also need to add a `Signed-off-by` line to please the DCO bot.

**Comment by mjeronimo** [comment]
@clalancette Ok, I'll update, thanks.

File: launch_testing_ros/package.xml [hunk 1]
```diff
@@ -4,6 +4,8 @@
   <name>launch_testing_ros</name>
   <version>0.10.2</version>
   <description>A package providing utilities for writing ROS2 enabled launch tests.</description>
+  <maintainer email="jacob@openrobotics.org">Jacob Perron</maintainer>
+  <maintainer email="michael.jeronimo@openrobotics.org">Michael Jeronimo</maintainer>
   <maintainer email="pete.baughman@apex.ai">Pete Baughman</maintainer>
```
Comment (clalancette) [inline launch_testing_ros/package.xml:9]: I'd go ahead and make Pete an `author` here.

**Review by clalancette — CHANGES_REQUESTED** [review]
One more minor thing to fix, then looks good.  Thanks.

**Comment by clalancette** [comment]
Generally looks good, but you'll have to go back and rebase and sign all of the commits for the DCO bot.

**Review by jacobperron — APPROVED** [review]

**Review by wjwwood — APPROVED** [review]

**Review by clalancette — APPROVED** [review]

**Pull request merged**